src/lib.rs

```diff
@@ -596,6 +596,7 @@ pub struct VariableField {
     pub unit: Option<Unit>,
     pub unit_exponent: Option<UnitExponent>,
     pub collections: Vec<Collection>,
+    pub is_relative: bool,
```

Comment:
I'm wondering if we should implement this like the `MainDataItem` trait in `hid.rs` where all these are implemented as `is_relative()` and `is_absolute()` etc. Because if we expose relative we should also expose the other bits like linear/nonlinear, etc (6.2.2.4 in the spec). What do you think?

Not sure anyone really cares about those but to have the complete implementation we should have them available.

Reply:
Can I punt this to you? :)